js/libs.js

```diff
@@ -445,6 +471,30 @@ Fliplet.Registry.set('notification-inbox:1.0:core', function(element, data) {
     }
   }
 
+  function showAllNotification() {
+    $('.notification-read, .notification-unread').show();
+  }
+
+  function showUnreadNotification() {
+    $('.notification-read').hide();
+    $('.notification-unread').show();
+  }
+
+  function changeToggleStatus(toggle) {
+    document.getElementById('notificationToggle').checked = toggle;
+    changeToggleText(toggle);
+  }
+
+  function changeToggleText(toggle) {
```

Comment:
I suggest adding event listener to the checkbox input. When this changes, add/remove a `checked` class to the `.switch-toggle-holder` container.

This allows you to use CSS to show and hide specific labels without having to use JavaScript to show/hide specific elements on the screen.